templates/todo/projects/python-mongo/.repo/terraform/infra/provider.tf

```diff
@@ -0,0 +1,30 @@
+#Set the terraform required version, and Configure the Azure Provider.Use remote storage

```

Comment:
I thought we were starting with local state storage; do we actually use remote storage?  I would have expected a `cloud` or `backend` block somewhere?

Reply:
The template should definitely use local storage.  We don't want to have to support remote state for our samples.

Reply:
the template uses local storage, fixed the comment